tests/test_cut_diff.py

```diff
@@ -242,6 +266,96 @@ def test_loading_edl(self):
                 else:
                     self.assertFalse(cut_diff.repeated)
 
+    def test_old_clip_for_shot(self):
+        """
+        Test that old_clip_for_shot retrieves the right clip.
+        """
+        self._add_sg_cut_data()
+
+        def get_clip_list():
+            prev_clip_list = []
+            for i in range(2):
+                clip = otio.schema.Clip(
+                    name="test_clip_%d" % i,
+                    source_range=TimeRange(
+                        RationalTime(10, 24),
+                        RationalTime(10, 24),  # duration, 10 frames.
+                    ),
+                    metadata={"sg": self.sg_cut_items[i]}
+                )
+                cut_clip = SGCutClip(clip, self.sg_shots[i], i + 1)
+                prev_clip_list.append(cut_clip)
+            return prev_clip_list
+        clip = otio.schema.Clip(
+            name="test_clip",
+            source_range=TimeRange(
+                RationalTime(10, 24),
+                RationalTime(10, 24),  # duration, 10 frames.
+            ),
+        )
+        # Test that if another_clip is not in prev_clip_list, it returns None
+        another_clip = SGCutClip(clip, self.sg_shots[3], 4)
+        prev_clip_list = get_clip_list()
+        old_clip = SGTrackDiff.old_clip_for_shot(another_clip, prev_clip_list, self.sg_shots[3])
```

Comment:
Is this the reason you turned it into a `staticmethod` to not have to instantiate a `SGTrackDiff`? I would really prefer it to be a `classmethod`. Later we might have some settings on the `class` or whatever.